src/Google/Ads/GoogleAds/Lib/V6/GoogleAdsCallLogger.php

```diff
@@ -65,6 +65,25 @@ public function __construct(
         }
     }
 
+    /**
+     * Logs summary and the details of the given status, request data and response.
+     *
+     * @param object $status the status to be logged
+     * @param array $requestData the request data
+     * @param object|null $response the response to be logged
+     */
+    public function log(object $status, array $requestData, ?object $response = null)
```

Comment:
I believe you are missing a `call` parameter to avoid the use of `$this`?

Reply:
Good call.
Fixed and confirmed that it worked.